README.md

```diff
@@ -21,8 +21,12 @@ Install this as your plugin's dependency:
 npm i vue-demi
 # or
 yarn add vue-demi
+# or 
+pnpm i vue-demi
 ```
 
+> When using pnpm, you will need to add `shamefully-hoist=true` to your project's `.npmrc` for `typescript` to find vue exports
```

Comment:
```suggestion
```

Reply:
@antfu why did you delete this warning? I created this pr to add this warning, as `Module '"vue-demi"' has no exported member 'App'.` Isn't an error one would know how to fix just by looking at it. I myself have been searching for solution of this issue for about a week.

People using npm alternative, that isn't preintalled with node usually know how to install packages, which makes the `pnpm i vue-demi` instruction unnecessary.

Reply:
It isn't a library's responsibility to explain the usage/caveat of each package manager and you can't really explain the complex idea well in a few lines. The solution you are providing isn't the only way that could solve the use (and it's probably a less optimal one from pnpm's perspective, given the `shamefully` prefix). Having this would make it misleading. IMO, when users chose a certain package manager, they should be responsible to read the docs and understand the strategies and caveats the package manager comes with.